### PR TITLE
feat: Add migration

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
This PR adds the migration command
Requires https://github.com/contentful/migration-cli/pull/90

TODO : 

- [x]  Add Unit tests
- [x] Release an Add migration-cli as dependency

### CODE REVIEW

**Comment by loudmouth** [comment]
Hey @Khaledgarbaya just wanted to give a heads up: i checked out this branch, built it and linked it, and i cannot succesfully run a migration (even when explicitly passing both the space id and environment id):

**My Syntax:**
```
contentful space migration --space-id $SPACE_ID --environment-id 'dev' migrations/01-add-category-field.js
```

**Output:**
```
/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/yargs.js:1079
      else throw err
           ^

Error: Cannot find file /Users/jpwright/contentful-custom-app/space.                                                                            
    at yargs.usage.demandCommand.check.args (/Users/jpwright/Contentful/js/CLI/contentful-cli/dist/cmds/space_cmds/migration.js:47:11)
    at Object.self.customChecks (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/lib/validation.js:212:18)
    at Object.Yargs.self._runValidation (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/yargs.js:1091:16)
    at Object.self.runCommand (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/lib/command.js:229:36)
    at Object.Yargs.self._parseArgs (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/yargs.js:990:30)
    at Object.self.runCommand (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/lib/command.js:204:45)
    at Object.Yargs.self._parseArgs (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/yargs.js:990:30)
    at Object.Yargs.self.parse (/Users/jpwright/Contentful/js/CLI/contentful-cli/node_modules/yargs/yargs.js:532:23)
    at Object.<anonymous> (/Users/jpwright/Contentful/js/CLI/contentful-cli/dist/cli.js:24:4)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)                                                                    
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
```

**Comment by Khaledgarbaya** [comment]
Hey @loudmouth, seems like it's dependency problem can you try to: 

1. `npm install`
2. `npm run build`
3. `npm link`
4. `contentful space migration --space-id $SPACE_ID --environment-id 'dev' migrations/01-add-category-field.js`

File: test/integration/cmds/space/snapshots/help.test.js.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Generated by [AVA](https://ava.li).
       export       export a space data to a json file␊
       import       import a space␊
       list         List your spaces                                    [aliases: ls]␊
+      migration    import a space␊
```
Comment (axe312ger) [inline test/integration/cmds/space/snapshots/help.test.js.md]: copy 🍝

File: test/integration/cmds/space/snapshots/help.test.js.md [hunk 2]
```diff
@@ -33,6 +34,7 @@ Generated by [AVA](https://ava.li).
...
```
Comment (axe312ger) [inline test/integration/cmds/space/snapshots/help.test.js.md]: copy 🍝

File: test/integration/cmds/space/snapshots/help.test.js.md [hunk 3]
```diff
@@ -51,6 +53,7 @@ Generated by [AVA](https://ava.li).
...
```
Comment (axe312ger) [inline test/integration/cmds/space/snapshots/help.test.js.md]: copy 🍝

**Pull request merged**